enterprise/admin/forms.py

```diff
@@ -333,7 +338,44 @@ class Meta:
             "enable_learner_portal",
             "learner_portal_hostname",
             "enable_portal_reporting_config_screen",
+            # "catalog_uuids",
+        )
+
+    def __init__(self, *args, **kwargs):
+        super(EnterpriseCustomerAdminForm, self).__init__(*args, **kwargs)
+        enterprise_customer = kwargs.get('instance', None)
+        self.fields["enterprise_customer_catalogs"].queryset = EnterpriseCustomerCatalog.objects.filter(
+            enterprise_customer=enterprise_customer
         )
+        self.fields["enterprise_customer_catalogs"].initial = enterprise_customer.customer_catalogs
```

Comment:
We should not set and initial value for this field it should be empty which means content of all catalogs needs to be transmitted.